test/components/how-to-play.test.js

```diff
@@ -75,7 +74,13 @@ describe("How To Play Screen", () => {
         howToPlayScreen.scene = { key: "test-select" };
         howToPlayScreen.game = { canvas: { parentElement: "parent-element" } };
         howToPlayScreen.navigation = { next: jest.fn() };
+        howToPlayScreen.sys = {
+            queueDepthSort: jest.fn(),
+        };
         howToPlayScreen.setLayout = jest.fn(() => mockLayout);
+        Object.defineProperty(howToPlayScreen, "layout", {
+            get: jest.fn(() => mockLayout),
```

Comment:
Not sure why this mock is needed? Will have a look...

Reply:
Seems like it isn't?

Reply:
Sorry - that should be line 77-79

Reply:
ah! Good spot. Was a fix for an intermediate issue I left behind. Updated :)